store/store.go

```diff
@@ -42,12 +42,27 @@ func (c *CloudSQLConnectorConfig) poolConfig(ctx context.Context) (*pgxpool.Conf
 	return pgxCfg, nil
 }
 
+type QueriesWithTX struct {
```

Comment:
I can't tell how awful this weird wrapper is, but the default `Queries` type created by `sqlc` does not include a method for starting a transaction, so I've shadowed their `WithTx` method with one that opens a Tx.